src/langs/fr.json

```
@@ -87,5 +87,7 @@
   "Add to Bookmarks": "Ajouter aux Signets",
   "Remove from Bookmarks": "Enlever des Signets",
   "Content": "Contenu",
-  "Type something....": "Ecrire quelque chose...."
+  "Type something....": "Ecrire quelque chose....",
+  "The element cannot be opened in the mobile app": "Cet {{type}} élément ne peut pas être ouvert dans l'application mobile",
+  "I would like to be able to see this item": "J'aimerais pouvoir voir cet article"
```

```suggestion
  "I would like to be able to see this item": "J'aimerais pouvoir voir cet élément"
```
but anyway the message will change (cf comment above)